### ISSUE TITLE
Install command truncated

### ISSUE THREAD

**Issue by Dean-Coakley** [helm/monocular#580]
From: https://github.com/helm/hub/issues/26

Expected:
![image](https://user-images.githubusercontent.com/15719653/49823367-64455100-fd77-11e8-9d92-43d7d1b85ceb.png)

Actual:
![image](https://user-images.githubusercontent.com/15719653/49823335-54c60800-fd77-11e8-95ff-73cbf4ef0ab9.png)

Fixed by removing:
```
/*.chart-details-usage .mat-input-element {
     width: calc(100% - 5em); 
}*/
```

I'd love to tackle this issue myself, but not sure if I am able to in the best way.

**Comment by prydonius** [comment]
Thanks for finding the fix @Dean-Coakley, I'd be happy to accept a PR from you to just remove the rule in https://github.com/helm/monocular/blob/master/frontend/src/app/chart-details/chart-details-usage/chart-details-usage.component.scss#L25. Let me know if you need any help with contributing!

**Comment by Dean-Coakley** [comment]
@prydonius Haven't been able to test this yet. Is there a nice way to test without the use of `telepresence`? It's causing issue for me.

**Comment by prydonius** [comment]
@Dean-Coakley hmm, what `telepresence` command are you running? You shouldn't be in the bitnami/nginx container, you should be inside a NodeJS container built from https://github.com/helm/monocular/blob/master/dev_env/ui/Dockerfile.

**Comment by Dean-Coakley** [comment]
@prydonius I ran 

`telepresence --swap-deployment dev-monocular-ui --namespace monocular --expose 4200:8080 --docker-run --rm -ti -v $(pwd)/frontend:/app quay.io/helmpack/monocular-ui bash` 

Following along: https://github.com/helm/monocular/blob/master/docs/development.md#starting-monocular-development-server

**Comment by prydonius** [comment]
@Dean-Coakley you need to first build the development container, see the `docker build` command above the `telepresence` command. The image used in the chart is different to the one that needs to be used in deployment.

**Comment by Dean-Coakley** [comment]
@prydonius Oh, apparently I was accidentally building the image outside of my minikube cluster and then trying to use it in the cluster. 😬 Thanks! 


Now I see the dev server but chart-svc is 404-ing so I can't see any charts UI.


Are there more prerequisites?
```
NAME                                              READY  STATUS            RESTARTS  AGE
dev-mongodb-7d55f9c6c5-pxvmq                      1/1    Running           1         7d
dev-monocular-chartsvc-9965dd946-62zxq            0/1    CrashLoopBackOff  15        7d
dev-monocular-chartsvc-9965dd946-rwmkv            0/1    CrashLoopBackOff  13        7d
dev-monocular-chartsvc-9965dd946-wmh8z            0/1    CrashLoopBackOff  13        7d
dev-monocular-prerender-8ffcff9cf-r9942           1/1    Running           1         7d
dev-monocular-ui-5996d58ff6-kn2xh                 1/1    Running           0         15m
dev-monocular-ui-5996d58ff6-m89bc                 1/1    Running           0         15m
dev-monocular-sync-initial-incubator-znumf-bc6nr  0/1    CrashLoopBackOff  7         14m
dev-monocular-sync-initial-stable-mpwky-826gx     0/1    CrashLoopBackOff  7         14m
```

```
$ kubectl logs dev-monocular-chartsvc-9965dd946-rwmkv -n monocular
 level=fatal msg="unable to connect to MongoDB" host=dev-mongodb
```

```
$ kubectl logs dev-mongodb-7d55f9c6c5-pxvmq -n monocular
2018-12-28T21:18:03.112+0000 I NETWORK  [listener] connection accepted from 172.17.0.13:45478 #295 (1 connection now open)
2018-12-28T21:18:03.112+0000 I NETWORK  [conn295] received client metadata from 172.17.0.13:45478 conn295: { driver: { name: "mgo", version: "globalsign" }, os: { type: "linux", architecture: "amd64" } }
2018-12-28T21:18:03.125+0000 I ACCESS   [conn295] SASL SCRAM-SHA-1 authentication failed for root on admin from client 172.17.0.13:45478 ; AuthenticationFailed: SCRAM authentication failed, storedKey mismatch
2018-12-28T21:18:03.125+0000 I NETWORK  [conn295] end connection 172.17.0.13:45478 (0 connections now open)
2018-12-28T21:18:11.115+0000 I NETWORK  [listener] connection accepted from 172.17.0.11:45796 #296 (1 connection now open)
2018-12-28T21:18:11.117+0000 I NETWORK  [conn296] received client metadata from 172.17.0.11:45796 conn296: { os: { type: "linux", architecture: "amd64" }, driver: { name: "mgo", version: "globalsign" } }
2018-12-28T21:18:11.127+0000 I ACCESS   [conn296] SASL SCRAM-SHA-1 authentication failed for root on admin from client 172.17.0.11:45796 ; AuthenticationFailed: SCRAM authentication failed, storedKey mismatch
2018-12-28T21:18:11.128+0000 I NETWORK  [conn296] end connection 172.17.0.11:45796 (0 connections now open)
```

Sorry for all the hassle!

**Comment by prydonius** [comment]
@Dean-Coakley No worries, sorry for the delay in responding! Have you run a `helm upgrade` on the chart? That might explain why chartsvc can't authenticate correctly with MongoDB, as `helm upgrade` will regenerate the secret for MongoDB (see https://github.com/helm/helm/issues/3053). To get both MongoDB and chartsvc to pick up the new password, you delete the MongoDB pod and all the chartsvc pods so they all start up again with the regenerated password. Hope that helps!

**Comment by Dean-Coakley** [comment]
Deleting all the pods seemed to work. But after the telepresence command I get into a blank container.
`$app# ls`

The telepresence command only works on terminal sessions **without** running `eval $(minikube docker-env)` beforehand.

I build my own image first with `$ docker build -t monocular_ui ./dev_env/ui`. I get into container with blank directories even when building the image from master.

**Comment by prydonius** [comment]
What directory are you running your telepresence command from? Make sure you're running it from the root for the volume mount to be correct, or change it as appropriate.

 `-v $(pwd)/frontend:/app` or  change it to `-v $(pwd):/app` if you're already in the frontend dir.

**Comment by Dean-Coakley** [comment]
@prydonius Yes! I was able to test and I'm happy with my changes! Thanks so much
